### ISSUE TITLE
1.1.11 unit test `dsp.3d.normal` fails

### ISSUE THREAD

**Issue by dvzrv** [sadko4u/lsp-plugins#62]
As mentioned in #61, I have (at least) one failing unit test when building and testing lsp-plugins 1.1.11.

The check log is attached:
[lsp-plugins-1.1.11-1-x86_64-check.log](https://github.com/sadko4u/lsp-plugins/files/3993056/lsp-plugins-1.1.11-1-x86_64-check.log)

**Comment by dvzrv** [comment]
A little note on how I run these tests:

```
make test
./.test/lsp-plugins-test utest \
    --nofork \
    --debug \
    --verbose
```

When running it without `--nofork` and `--debug` I don't get failing tests though:
[lsp-plugins-1.1.11-check-avx2.log](https://github.com/sadko4u/lsp-plugins/files/3993073/lsp-plugins-1.1.11-check-avx2.log)

**Comment by sadko4u** [comment]
Please try to build and test from the **github-issue-61** branch.

**Comment by dvzrv** [comment]
Thank you! The branch fixes the tests for me!

**Comment by sadko4u** [comment]
This is the rare case when native implementation was wrong but vectorized implementation was correct.

**Comment by sadko4u** [comment]
Merged changes into **devel** branch: 1bccb10dc5d7906795a7b96f6a76d31e3f3f0f4d

**Comment by sadko4u** [comment]
Available in 1.1.13 release!
